examples/plugins/http-requests/README.md

```diff
@@ -0,0 +1,89 @@
+# HTTP requests plugin
+
+The HTTP requests plugin tracks the number of incoming HTTP requests per process over time using eBPF.
+The UI shows the number of requests as a single value and graph which are updated every second.
+The HTTP requests plugin does not use any application-level instrumentation, it has a negligible performance toll (metrics are obtained in-kernel without any packet copying to userspace).
+
+**Note:** This plugin needs a [recent kernel version with eBPF support](https://github.com/iovisor/bcc/blob/master/INSTALL.md#kernel-configuration).
```

Comment:
I have another PR almost ready about the main plugin README. I will do it there if it is ok for you.